src/RunTests/Microsoft.Build.RunTests.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0</TargetFrameworks>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+		<PackageId>Microsoft.Build.RunTests</PackageId>
+		<Description>Runs VS Test Console</Description>
+		<NoDefaultExcludes>true</NoDefaultExcludes>
+		<BuildOutputTargetFolder>build\</BuildOutputTargetFolder>
+		<AppendTargetFrameworkToOutputPath>true</AppendTargetFrameworkToOutputPath>
+		<DevelopmentDependency>true</DevelopmentDependency>
+		<LangVersion>latest</LangVersion>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+	</PropertyGroup>
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Build.Utilities.Core" />
+		<PackageReference Include="Microsoft.TestPlatform" />
```

Comment:
I wonder if we need it, means take always this deps also if in future we add new runners

Reply:
Are you referring to needing the Microsoft.TestPlatform?